src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
-            [EPathType.EPathTypeTableIndex]: undefined,
-            [EPathType.EPathTypeExtSubDomain]: undefined,
-            [EPathType.EPathTypeColumnStore]: undefined,
-            [EPathType.EPathTypeColumnTable]: undefined,
-            [EPathType.EPathTypeCdcStream]: () => <CDCStreamOverview data={currentObjectData} />,
-            [EPathType.EPathTypePersQueueGroup]: () => (
-                <PersQueueGroupOverview data={currentObjectData} />
-            ),
-            [EPathType.EPathTypeExternalTable]: () => (
-                <ExternalTableSummary data={currentObjectData} />
-            ),
-            [EPathType.EPathTypeExternalDataSource]: () => (
-                <ExternalDataSourceSummary data={currentObjectData} />
-            ),
-            [EPathType.EPathTypeView]: undefined,
-            [EPathType.EPathTypeReplication]: () => (
-                <InfoViewer
-                    info={[
-                        {
-                            label: createdAtLabel,
-                            value: createdAt,
-                        },
-                        {
-                            label: 'State',
-                            value: (
-                                <AsyncReplicationState
-                                    state={
-                                        currentObjectData?.PathDescription?.ReplicationDescription
-                                            ?.State
-                                    }
-                                />
-                            ),
-                        },
-                    ]}
-                />
-            ),
-        };
+        if (!currentSchemaData) return undefined;
+        const {CreateStep, PathType, PathSubType, PathId, PathVersion} = currentSchemaData;
+
+        const overview: InfoViewerItem[] = [];
+
+        // for any schema type
+        overview.push({label: 'Type', value: PathType?.replace(/^EPathType/, '')});
 
-        let component =
-            currentSchemaData?.PathType && pathTypeToComponent[currentSchemaData.PathType]?.();
+        // show SubType if it's not EPathSubTypeEmpty
+        if (PathSubType !== EPathSubType.EPathSubTypeEmpty) {
+            overview.push({label: 'SubType', value: PathSubType?.replace(/^EPathSubType/, '')});
+        }
+
+        // show PathId
+        // if you get 18446744073709551615 = ffffffff = 2n ** 64n - 1n =-1 в ui64
+        // you need to ask @xeno0904 for help
+        overview.push({label: 'Id', value: PathId});
 
-        if (!component) {
-            component = <InfoViewer info={[{label: createdAtLabel, value: createdAt}]} />;
+        // show PathVersion
+        overview.push({label: 'Version', value: PathVersion});
+
+        // show created time if it's not 0
+        if (isNumeric(CreateStep) && Number(CreateStep)) {
+            overview.push({
+                label: 'Created',
+                value: dateTimeParse(Number(CreateStep))?.format('YYYY-MM-DD HH:mm'),
```

Comment:
let's add helper in `src/utils` to format dates in one place. This helper will be used here and in `formatDateTime` function.